addons/mrp/views/mrp_routing_views.xml

```diff
@@ -54,7 +54,9 @@
             <field name="arch" type="xml">
                 <xpath expr="//tree" position="attributes">
                     <attribute name="create">0</attribute>
+                    <attribute name="delete">0</attribute>
                     <attribute name="export_xlsx">0</attribute>
+                    <attribute name="multi_edit">0</attribute>
```

Comment:
Is it useful? Seems False by default and basic view doesn't set it

Reply:
The inherited view has it active. If we don't explicitly deactivate it in this view then you can multi-edit but not individual edit (plus it would be weird for the view to allow editing at all).